src/publisher/index.ts

```diff
@@ -264,9 +256,23 @@ export class Publisher {
   constructSpan(message: PubsubMessage): Span | undefined {
     const spanAttributes = {
       data: message.data,
```

Comment:
@feywind I just noticed this was already here but probably shouldn't be written to cloud trace?

Reply:
Awaiting feedback from @feywind. Personally, I think we should remove it as it could potentially unexpectedly leak unwanted PPI information to the service the traces are exported too

Reply:
It also might end up being a lot of wasted data, if the messages are on the bigger side, so yeah, might be good to take that part out.

Reply:
Removed it with a BREAKING CHANGE marker in the commit